### ISSUE TITLE
Wrap parameters removing leading spacing

### ISSUE THREAD

**Issue by jshier** [nicklockwood/SwiftFormat#1299]
With `--wrapparameters after-first` or `preserve`, the following:

```swift
let loginResponse = LoginResponse(status: .success(.init(accessToken: session,
                                                         status: .enabled)),
                                  invoicingURL: .invoicing,
                                  paymentFormURL: .paymentForm)
```

is transformed into this:

```swift
let loginResponse = LoginResponse(status: .success(.init(accessToken: session,
                                                         status: .enabled)),
invoicingURL: .invoicing,
paymentFormURL: .paymentForm)
```

This seems wrong for both settings. I'd like to use `preserve`, and it usually does, but this structure seems to break it.

**Comment by nicklockwood** [comment]
@jshier working on this now. It seems to be a bug with the `indent` rule. Disabling indent for this file or code section is a workaround.

**Comment by nicklockwood** [comment]
Fixed in 0.50.7
